docs/metrics/prometheus/grafana/minio-dashboard.json

```
@@ -2326,6 +2326,110 @@
         "align": false
       }
     },
+    {
+      "aliasColors": {
+        "10.13.1.25:9000 DELETE": "red",
+        "10.13.1.25:9000 GET": "green",
+        "10.13.1.25:9000 POST": "blue"
+      },
+      "bars": true,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "oQpdp674k"
+      },
+      "description": "Monitor TCP dial timeout/errors detected by nodes",
```

```suggestion
      "description": "Total TCP dial timeout/errors detected by nodes",
```

This is already monitoring we don't have say that its monitoring again.